config/default.json

```diff
@@ -69,10 +126,20 @@
     "tasksBatchSize": 5
   },
   "validationValuesByInfo": {
-    "crs":[4326],
-    "fileFormat":["GPKG"],
-    "pixelSizeRange" : { "min": 0.000000335276,"max": 0.703125 },
-    "tileSize" : 256
+    "crs": [
+      4326
+    ],
+    "fileFormat": [
+      "GPKG"
+    ],
+    "pixelSizeRange": {
+      "min": 0.000000335276,
+      "max": 0.703125
+    },
+    "tileSize": 256
   },
-  "forbiddenTypesForParallelIngesion": ["Ingestion_New", "Ingestion_Update"]
-}
+  "forbiddenTypesForParallelIngesion": [
+    "Ingestion_New",
+    "Ingestion_Update"
+  ]
+}
```

Comment:
end of file
consider to disable lint

Reply:
I'd checked, and the linter doesn't affect on config, the reason for struct change was because i was manually formated the document on vc-code